tests/BlazingSQLTest/EndToEndTests/TestSuites/bindableAliasTest.yaml

```diff
@@ -0,0 +1,113 @@
+#Bindable Alias Test Suite
+
+TEST_SUITE:
+  SETUP:
+    APPLY_ORDER: True
+    USE_PERCENTAGE: False
+    ACCEPTABLE_DIFFERENCE: 0.01
+    COMPARE_WITH: drill
+    SKIP_WITH: CUDF
```

Comment:
the SKIP_WITH parameters should be more explicit like:
CUDF_MULTI-NODE

Reply:
Done